app/events.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+  If space or enter is pressed while focused on an element then click that
+  element.
+*/
+function clickOnSpaceOrEnter(event: KeyboardEvent) {
```

Comment:
I'm surprised there isn't more built-in accessibility features like this

Reply:
I think its default built into `button` elements, but not `div`s like we are using. Esri also uses keydown events for these things on their buttons I think.